FireBrick/UI/TeamViewController/TeamViewController.swift

```diff
@@ -0,0 +1,359 @@
+//
+//  TeamViewController.swift
+//  FireBrick
+//
+//  Created by metoSimka on 28/05/2019.
+//  Copyright © 2019 metoSimka. All rights reserved.
+//
+
+import UIKit
+import Firebase
+import FirebaseFirestore
+import FirebaseStorage
+import SwiftEntryKit
+
+class TeamViewController: UIViewController {
```

Comment:
use https://gist.github.com/kryndach/c4a8d07cdbff97f747b8f9dfbf87209b for code structure
this need to be done for all classes